spec/models/flag_spec.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 RSpec.describe Flag, type: :model do
-  before { Flag.reset_flag_settings! }
+  around do |test|
```

Comment:
Why not do `before` and `after` instead of around? Just curious, doesn't need to change necessarily.

Reply:
no valid reason here :)